packages/crawler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "test:e2e:puppeteer": "jest --colors -c tests/e2e/config/jest.config.ts",
+    "test:e2e": "concurrently -k -s 'first' -n 'cdn,puppeteer' 'serve-cdn' 'yarn test:e2e:puppeteer'"
```

Comment:
`clean` commands are missing and should be called from root `package.json`, too.

Reply:
Fixed